client/ayon_core/plugins/publish/collect_udim.py

```diff
@@ -0,0 +1,41 @@
+"""Publish UDIM tiles."""
+import re
+
+import pyblish.api
+from ayon_core.lib import BoolDef
+from ayon_core.pipeline import publish
+
+UDIM_REGEX = re.compile(r"(.*)\.(?P<udim>\d{4})\.(.*)")
+
+
+class CollectUDIMs(
+    pyblish.api.InstancePlugin, publish.AYONPyblishPluginMixin):
+    """Collect UDIMs tiles."""
+
+    label = "Collect UDIMs"
+    order = pyblish.api.CollectorOrder + 0.499
+    families = ["image"]
+
+    def process(self, instance):
+        # type: (pyblish.api.Instance) -> None
+        instance_settings = self.get_attr_values_from_data(instance.data)
+        is_udim = instance_settings.get("isUDIM", False)
+        if not is_udim:
+            return
+
+        for representation in instance.data["representations"]:
+            if isinstance(representation["files"], (list, tuple)):
+                continue
```

Comment:
This is wrong, no?

Wouldn't this technically _actually_ skip sequences of UDIMs?
Also, should we pop `frame` from the `representation`

Reply:
I think Libor's screenshot show the problem - it's generating a representation PER image in the sequence? Even though it should be one representation.